cooltools/api/snipping.py

```diff
@@ -696,22 +871,8 @@ def pileup(
     else:
         raise ValueError("Unknown feature_df format")
 
-    if feature_type == "bed":
-        features_df = bioframe.assign_view(
-            features_df, view_df, df_view_col="region", view_name_col=view_name_col,
-        )
-    else:
-        features_df = assign_view_paired(
-            features_df, view_df, view_name_col=view_name_col
-        )
-        # Now we consolidate the region annotations into one column `region`
-        # Features that cross between regions get region `None`
-        # The rest get the `region1` value assigned to a new column `region`
-        features_df["region"] = np.where(
-            features_df["region1"] == features_df["region1"],
-            features_df["region1"],
-            None,
-        )
+    features_df = assign_regions(features_df, view_df)
```

Comment:
yessss

Reply:
Or not... wtf, idk what's going on

Reply:
Anyway, the plan as to deprecate this assign_regions and use bioframe, but it is shorter like this...